stable/kong/templates/migrations-pre-upgrade.yaml

```diff
@@ -68,6 +68,6 @@ spec:
         - name: KONG_CASSANDRA_CONTACT_POINTS
           value: {{ template "kong.cassandra.fullname" . }}
         {{- end }}
-        command: [ "/bin/sh", "-c", "kong migrations up" ]
+        command: [ "/bin/sh", "-c", "kong migrations bootstrap" ]
```

Comment:
This will work with 0.15.0 but it will break for a migration from `1.0` to `1.1`.
Bootstrap needs to be done only once and not upon every migration.

Reply:
If you can make this version aware of when to use `up` and when to use `bootstrap`, it can be merged in.